.gitignore

```diff
@@ -4,6 +4,8 @@ temp-spec/
 node_modules/
 out/
 coverage/
-dist/yoast-seo*
+dist/
+js/
```

Comment:
Because we're now publishing on NPM, and we're going to direct people to using that instead of linking the github repo, the js folder can be removed. Please make sure `grunt build` on YoastSEO.js is not broken after removing the js folder.